### PR TITLE
docs: additional postinstall guides

### PR DESCRIPTION
**Pull request by secretmango**
reordered the GRUB password guide, to show the "dont worry" message earlier and the command directly after the "here is the command" message, improving clarity.

fixed incorrect polkit guide, following a recent Fedora Change request. Also removed incorrect "flatpak" group which does not do anything. This would be a group that could be added manually.

added a guide on how to add an SELinux confined user, linking to upstream docs and avoiding duplication. #338

added a guide on how to use QEMU/libvirt without root, via the unprivileged QEMU user session

### CODE REVIEW

File: POSTINSTALL-README.md [hunk 1]
```diff
@@ -48,7 +48,9 @@ Documentation is available [here](https://github.com/secureblue/secureblue/blob/
 
 Setting a GRUB password helps protect the device from physical tampering and mitigates various attack vectors, such as booting from malicious media devices and changing boot or kernel parameters.
 
-To set a GRUB password, use the following command. By default, the password will be required when modifying boot entries, but not when booting existing entries.
```
Comment (RoyalOughtness) [inline POSTINSTALL-README.md:51]: This seems fine how it was

File: POSTINSTALL-README.md [hunk 2]
```diff
@@ -66,9 +68,9 @@ Creating a dedicated wheel user and removing wheel from your primary user helps
 > [!CAUTION]
 > If you do these steps out of order, it is possible to end up without the ability to administrate your system. You will not be able to use the [traditional GRUB-based method](https://linuxconfig.org/recover-reset-forgotten-linux-root-password) of fixing mistakes like this, either, as this will leave your system in a broken state. However, simply rolling back to an older snapshot of your system, should resolve the problem.
 
-1. ```adduser admin```
-2. ```usermod -aG wheel admin```
-3. ```passwd admin```
+1. ```sudo adduser admin```
```
Comment (RoyalOughtness) [inline POSTINSTALL-README.md]: sudo is implied, this isn't really necessary.

File: POSTINSTALL-README.md [hunk 3]
```diff
@@ -117,3 +124,22 @@ The custom rule allows the group`diskadmin` to do the actions for unlocking and
...
+As this may cause breakages, [follow these guides](https://fedoraproject.org/wiki/SELinux/ConfinedUsers#Assign_a_SELinux_user_to_an_existing_Linux_user) on how to add a new user account and confine it.
+
+First, create a new user, using the above guide on adding a dedicated admin user. You can leave out the `wheel` group if you like.
+
+SELinux confined users have different access levels. From most privileged to least privileged:
+- `sysadm_u`
+- `staff_u`
+- `user_u`
+
+## Use a virtual machine without root
```
Comment (RoyalOughtness) [inline POSTINSTALL-README.md]: this is not a postinstall step, maybe an FAQ item.

File: POSTINSTALL-README.md [hunk 4]
```diff
@@ -117,3 +124,22 @@ The custom rule allows the group`diskadmin` to do the actions for unlocking and
 ## Instruction set optimizations for hardened_malloc
 
 Please see the description for release [v2.2.0](https://github.com/secureblue/secureblue/releases/tag/v2.2.0)
+
+## Add a SELinux confined user
+[SELinux confined users](https://fedoraproject.org/wiki/SELinux/ConfinedUsers) are a project to secure not only the core system, but also the user accounts with [SELinux](https://docs.fedoraproject.org/en-US/quick-docs/selinux-getting-started/).
+
+At the current state, unlike on Android, a user account and all it's processes are not protected by SELinux on Fedora.
+
+As this may cause breakages, [follow these guides](https://fedoraproject.org/wiki/SELinux/ConfinedUsers#Assign_a_SELinux_user_to_an_existing_Linux_user) on how to add a new user account and confine it.
```
Comment (RoyalOughtness) [inline POSTINSTALL-README.md]: We need to incorporate these steps into the guide for user provisioning instead of just linking to the fedora docs. it should be one seamless set of steps

**Review by RoyalOughtness — CHANGES_REQUESTED** [review]

File: POSTINSTALL-README.md (hunk 2 above)
Comment (secretmango) [reply POSTINSTALL-README.md]: then the commands below are not correct. Also, the 4th command works without sudo.

I also prefer highly if given commands work. That way, it would be a barrier to users "to know that stuff". This includes knowing that `reboot` is `systemctl reboot` which is done via polkit and doesnt need sudo... Which I wouldnt think users need to know.

File: POSTINSTALL-README.md (hunk 1 above)
Comment (secretmango) [reply POSTINSTALL-README.md:51]: if you think so. If I "resolve" this, will it be changed automatically?

File: POSTINSTALL-README.md (hunk 3 above)
Comment (secretmango) [reply POSTINSTALL-README.md]: okay, will add an FAQ instead

File: POSTINSTALL-README.md (hunk 4 above)
Comment (secretmango) [reply POSTINSTALL-README.md]: okay, I wanted to avoid duplication and make it more maintainable that way. The link is also directly to the steps needed, the other step (creating the user) is included there.

So this should not be in the FAQ but stay here?

**Comment by secretmango** [comment]
I added some updates!

File: POSTINSTALL-README.md (hunk 1 above)
Comment (RoyalOughtness) [reply POSTINSTALL-README.md:51]: no

File: POSTINSTALL-README.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# secureblue
+# Secureblue Post Install Recommendations
```
Comment (RoyalOughtness) [inline POSTINSTALL-README.md:1]: this was fine how it was

File: POSTINSTALL-README.md [hunk 6]
```diff
@@ -78,35 +81,33 @@ Creating a dedicated wheel user and removing wheel from your primary user helps
...
+- `sysadm_u`
+- `staff_u`
+- `user_u`
+
+Example for adding a user with the loose `sysadm_u` protection
 
 ```
-cat >> /etc/polkit-1/rules.d/80-udisks2.rules <<EOF
-polkit.addRule(function(action, subject) {
-    if (action.id == "org.freedesktop.udisks2.encrypted-unlock-system" || action.id == "org.freedesktop.udisks2.filesystem-mount-system" &&
```
Comment (RoyalOughtness) [inline POSTINSTALL-README.md:98]: why was this removed?

Comment (RoyalOughtness) [reply POSTINSTALL-README.md:98]: the comment I made was just about VMs :)

**Comment by RoyalOughtness** [comment]
https://github.com/secureblue/secureblue/issues/338#issuecomment-2259311313
